extension/test/unit/cancel-or-refund.handler.spec.js

```diff
@@ -0,0 +1,122 @@
+const { cloneDeep } = require('lodash')
+const nock = require('nock')
+const { expect } = require('chai')
+const configLoader = require('../../src/config/config')
+const { CTP_INTERACTION_TYPE_CANCEL_OR_REFUND } = require('../../src/config/constants')
+const { execute } = require('../../src/paymentHandler/cancel-or-refund.handler')
+
+describe('cancel-or-refund.handler::execute::', () => {
+  const cancelOrRefundResponse = {
+    pspReference: '853589969905798D',
+    response: '[cancelOrRefund-received]'
+  }
+  /* eslint-disable*/
+  const authorisedPayment = {
+    transactions: [
+      {
+        id: 'transaction1',
+        type: 'Authorization',
+        interactionId: '883589969904820D',
+        state: 'Success'
+      }
+    ],
+    interfaceInteractions: [
+      {
+        type: {
+          typeId: 'type',
+        },
+        fields: {
+          type: 'makePayment',
+          request: '{\"amount\":{\"currency\":\"EUR\",\"value\":1000},\"reference\":\"YOUR_ORDER_NUMBER\",\"paymentMethod\":{\"type\":\"scheme\",\"encryptedCardNumber\":\"test_4111111111111111\",\"encryptedExpiryMonth\":\"test_03\",\"encryptedExpiryYear\":\"test_2030\",\"encryptedSecurityCode\":\"test_737\"},\"returnUrl\":\"https://your-company.com/\",\"merchantAccount\":\"CommercetoolsGmbHDE775\"}',
+          response: '{\"pspReference\":\"883589969904820D\",\"resultCode\":\"Authorised\",\"amount\":{\"currency\":\"EUR\",\"value\":1000},\"merchantReference\":\"YOUR_ORDER_NUMBER\"}',
+          createdAt: '2020-05-20T10:18:25.073Z'
+        }
+      }
+    ]
+  }
+  /* eslint-enable */
+
+  let scope
+  const config = configLoader.load()
+  beforeEach(() => {
+    scope = nock(`${config.adyen.legacyApiBaseUrl}`)
+  })
+
+  it('given a payment with a type "Refund" transaction, '
+    + 'when /cancelOrRefund" request to Adyen is issued successfully '
+    + 'then it should return actions '
+    + '"addInterfaceInteraction", "changeTransactionState" and "changeTransactionInteractionId"', async () => {
+    scope.post('/cancelOrRefund').reply(200, cancelOrRefundResponse)
+
+    const paymentObject = cloneDeep(authorisedPayment)
+    paymentObject.transactions.push({
+      id: 'transaction2',
+      type: 'Refund',
+      state: 'Initial'
+    })
+
+    const response = await execute(paymentObject)
+    assertUpdateActions(response)
+  })
+
+  function assertUpdateActions ({ actions }) {
```

Comment:
Can you put this helper method after all the tests?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/286/commits/01d482cc780f0bc23bbc5cd0ecc58abade49d5c1